sig-feature-store/charter.md

```diff
@@ -0,0 +1,42 @@
+# SIG Feature Store Charter
+
+This charter adheres to the conventions, roles and organization management outlined in [wg-governance].
+
+## Scope
+
+SIG Feature Store covers the definition, management, storage, discovery, and serving of features to models.
+
+### In scope
+
+#### Code, Binaries and Services
```

Comment:
@woop I would suggest rewording this section a bit. The SIG doesn't technically own these artifacts since they are being developed outside of Kubeflow; eg. in Feast.

I would define the scope as more along the lines of coordinating between the various projects using or depending on feature stores.

Reply:
I've taken a stab at rewording it. Mind taking another look?

Reply:
https://github.com/kubeflow/community/pull/408/commits/fc86d5c151eb28908255020b8dbef94100abf6cf